freqtrade/misc.py

```diff
@@ -40,7 +40,7 @@ def file_dump_json(filename: Path, data: Any, is_zip: bool = False, log: bool =
         if log:
             logger.info(f'dumping json to "{filename}"')
         with filename.open("w") as fp:
-            rapidjson.dump(data, fp, default=str, number_mode=rapidjson.NM_NATIVE)
+            rapidjson.dump(data, fp, default=str, number_mode=rapidjson.NM_NATIVE, indent=4)
```

Comment:
what's the particular reason (which file exactly)?

This is used quite widely throughout the bot - so i don't think this will be a good idea (it might even break a few things - that has to be investigated) - as it'll increase the amount of memory used to store one file (and with that, also the latency for that) by quite a lot.

Reply:
after backtesting, output was stored in user_data/backtest_results/*.json
it's hard to read because there is no indent

Reply:
I got your idea.

Reply:
you can use `freqtrade backtesting-show` to show the last result again - or use one of the data analysis approaches to look at it in detail.

at any rate, it's not meant to be read manually.
If you really must - you can open it in vscode and use "format document".

it'll increase the size of the file by a huge amount - so not going to accept this, sorry.